src/data/archive/winter2024/stackschool.yml

```diff
@@ -0,0 +1,66 @@
+- name: StackSchool
+  quarter: Winter 2024
+  mainLink: https://github.com/uclaacm/stackschool-running_demo-w24 
+  tags: ["fullstack", "mern", "mongodb", "express", "reactjs", "nodejs"]
+  directors:
+    - Shiyu Ye
+    - Jenna Wang
+
+  workshops:
+    - name: "Session 1: Intro to Fullstack Mobile Development"
+      repo: https://github.com/uclaacm/stackschool-running_demo-w24/tree/main/session-1
+      slides: https://docs.google.com/presentation/d/11Ix6lqOcADqmuPp3QULNyEie8cY0QGNmnyuTbkLSOOs/edit 
+      youtube: https://www.youtube.com/watch?v=eahauN5074c&list=PLPO7_kXilXFYg-9u3t_GpvazKlqPd2j0h&ab_channel=ACMatUCLA 
+      tags: ["fullstack"]
+      presenters:
+        - Shiyu Ye
+
+    - name: "Session 2: Frontend"
+      repo: https://github.com/uclaacm/stackschool-running_demo-w24/tree/main/session-2 
+      slides: https://docs.google.com/presentation/d/1DSyVK3tAb3RHQA0BipLtSfiQtvq4fyTHmAlsjxbBhQw/edit#slide=id.g1cf0e2b9972_0_0
+      youtube: https://www.youtube.com/watch?v=Na0s4emaMFs&t=7s&ab_channel=ACMatUCLA
+      tags: ["frontend", "react native"]
+      presenters:
+        - James Wu 
+        - Max Lee 
+
+    - name: "Session 3: SQL"
+      repo: https://github.com/uclaacm/stackschool-running_demo-w24/tree/main/session-3 
+      slides: https://docs.google.com/presentation/d/1wFKMfRDAd98FNqJZCmnV8iw7TJ8N7H11FSyzjaxouLM/edit#slide=id.g1cf0e2b9972_0_0
+      youtube: https://www.youtube.com/watch?v=UbCIrLP6qBA&ab_channel=ACMatUCLA 
+      tags: ["backend", "sql", "database"]
+      presenters:
+        - Sneha Agarwal
+        - Nathan Zhang
+
+    - name: "Session 4: Servers"
+      repo: https://github.com/uclaacm/stackschool-running_demo-w24/tree/main/session-4 
+      slides: https://docs.google.com/presentation/d/1EP4b4GhPegNu_2VdUl9TXSDs8cwSPcwXPxVoMM65K1I/edit#slide=id.g2b5e276afe9_0_43
+      youtube: https://www.youtube.com/watch?v=88siNLQsIqg&list=PLPO7_kXilXFYg-9u3t_GpvazKlqPd2j0h&index=4&ab_channel=ACMatUCLA
+      tags: ["backend", "nodejs"]
+      presenters:
+        - Anakin Trotter
+        - Kayla Hamakawa
+    
+    - name: "Session 5: Backend Integration"
+      repo: https://github.com/uclaacm/stackschool-running_demo-w24/tree/main/session-5
+      slides: https://docs.google.com/presentation/d/1scnBk7kcKiTpBJjeSSCohgKtjVYsNcMyyCW69sotG0Y/edit#slide=id.g2b6396ae016_0_133
```

Comment:
I think the slides for this link to slide 2 instead of 1. Everything else looks good though-- do you think you could change this, and then I'll go ahead and merge it into main? Thanks!